spring-cloud-circuitbreaker-resilience4j/src/main/java/org/springframework/cloud/circuitbreaker/resilience4j/Resilience4JCircuitBreaker.java

```diff
@@ -70,14 +73,28 @@ public Resilience4JCircuitBreaker(String id,
 		this.circuitBreakerCustomizer = circuitBreakerCustomizer;
 	}
 
+	@Deprecated
 	public Resilience4JCircuitBreaker(String id,
 			io.github.resilience4j.circuitbreaker.CircuitBreakerConfig circuitBreakerConfig,
 			TimeLimiterConfig timeLimiterConfig,
 			CircuitBreakerRegistry circuitBreakerRegistry,
 			TimeLimiterRegistry timeLimiterRegistry, ExecutorService executorService,
 			Optional<Customizer<io.github.resilience4j.circuitbreaker.CircuitBreaker>> circuitBreakerCustomizer,
 			Resilience4jBulkheadProvider bulkheadProvider) {
+		this(id, id, circuitBreakerConfig, timeLimiterConfig, circuitBreakerRegistry, timeLimiterRegistry, executorService,
+			circuitBreakerCustomizer, bulkheadProvider);
+	}
+
+	public Resilience4JCircuitBreaker(String id,
+									  String groupName,
+									  io.github.resilience4j.circuitbreaker.CircuitBreakerConfig circuitBreakerConfig,
+									  TimeLimiterConfig timeLimiterConfig,
+									  CircuitBreakerRegistry circuitBreakerRegistry,
+									  TimeLimiterRegistry timeLimiterRegistry, ExecutorService executorService,
+									  Optional<Customizer<io.github.resilience4j.circuitbreaker.CircuitBreaker>> circuitBreakerCustomizer,
+									  Resilience4jBulkheadProvider bulkheadProvider) {
 		this.id = id;
+		this.groupName = groupName;
```

Comment:
Can this be leveraged in the non-bulkhead case?  Maybe as a tag?

Reply:
Added tags with group